admin/configuration.php

```diff
@@ -161,7 +162,7 @@
                   ?>
                   <?php
                   // multilanguage support:
-                  // For example, in admin/includes/languages/spanish/configuration.php
+                  // For example, in admin/includes/languages/spanish/lang.configuration.php
```

Comment:
Since this PR is probably going to be open for awhile longer, you are welcome to move this typo-correction (and the other one on line 61) to a separate PR, which we'll merge right away.
I don't mind doing it myself, but it will create a merge-conflict for you to have to fix anyway.

Reply:
No problem, it is done.